### PR TITLE
[path_distance_calculator]add readme

### PR DESCRIPTION
**Pull request by tkimura4**
## Description

add readme of path distance calculator 
## Review Procedure
Check the readme

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [ ] Read [pull request guidelines][pull-request-guidelines]
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR / build-and-test-pr**: Required to pass before the merge.
- **Build and test for PR / clang-tidy-pr**: NOT required to pass before the merge. It is up to the reviewer(s). Found false positives? See the [guidelines][clang-tidy-guidelines].
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[clang-tidy-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/ClangTidyGuideline/
[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[pull-request-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/PullRequestGuideline/
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

File: common/util/path_distance_calculator/Readme.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+### Output
+
+| Name         | Type                                       | Description                                                                                         |
+| ------------ | ------------------------------------------ | --------------------------------------------------------------------------------------------------- |
+| `~/distance` | `autoware_debug_msgs::msg::Float64Stamped` | Publish a distance from the path point closest to the self-position to the end point of the path[m] |
+
+## Parameters
+
+### Node Parameters
```
Comment (taikitanaka3) [inline common/util/path_distance_calculator/Readme.md:27]: @tkimura4 
parameterがない場合は記述しなくても良さそうですが、どうでしょう？cc : @kenji-miyake

Comment (kenji-miyake) [reply common/util/path_distance_calculator/Readme.md:27]: 表は無しで、Noneとか書いておけば良いんじゃないですかね。セクションはあって良いと思います。

File: common/util/path_distance_calculator/Readme.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# autoware_state_monitor
```
Comment (kenji-miyake) [inline common/util/path_distance_calculator/Readme.md:1]: :eye: :eye:

File: common/util/path_distance_calculator/Readme.md (hunk 1 above)
Comment (tkimura4) [reply common/util/path_distance_calculator/Readme.md:27]: Fixed at d2b33f6

File: common/util/path_distance_calculator/Readme.md (hunk 2 above)
Comment (tkimura4) [reply common/util/path_distance_calculator/Readme.md]: !!!!!!!!!!
Fixed at d2b33f6

File: common/util/path_distance_calculator/Readme.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+## Purpose
+
+This node publishes a distance from the path point closest to the self-position to the end point of the path.
```
[the remark this answers is not in the document]
Comment (tkimura4) [reply common/util/path_distance_calculator/Readme.md]: 72c4d96

File: common/util/path_distance_calculator/Readme.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+| Name                                                              | Type                                     | Description    |
+| ----------------------------------------------------------------- | ---------------------------------------- | -------------- |
+| `/planning/scenario_planning/lane_driving/behavior_planning/path` | `autoware_auto_planning_msgs::msg::Path` | Reference path |
+| `/tf`                                                             | `tf2_msgs/TFMessage`                     | TF (self-pose) |
+
+### Output
+
+| Name         | Type                                       | Description                                                                                         |
+| ------------ | ------------------------------------------ | --------------------------------------------------------------------------------------------------- |
+| `~/distance` | `autoware_debug_msgs::msg::Float64Stamped` | Publish a distance from the path point closest to the self-position to the end point of the path[m] |
```
[the remark this answers is not in the document]
Comment (tkimura4) [reply common/util/path_distance_calculator/Readme.md]: 72c4d96

**Review by taikitanaka3 — APPROVED** [review]

**Pull request merged**